docs/.benchmarks_table.md

```diff
@@ -2,11 +2,11 @@
 
 Package | Version | Relative Performance | Mean validation time
 --- | --- | --- | ---
-attrs + cattrs | `19.3.0` |  | 11.0μs
-valideer | `0.4.2` | 1.5x slower | 16.9μs
-trafaret | `2.0.2` | 5.7x slower | 63.3μs
-pydantic | `1.5.1` | 6.7x slower | 73.3μs
-voluptuous | `0.11.7` | 7.7x slower | 85.0μs
-marshmallow | `3.6.0` | 11.2x slower | 123.2μs
-django-rest-framework | `3.11.0` | 59.7x slower | 657.4μs
-cerberus | `1.3.2` | 129.8x slower | 1429.2μs
+attrs + cattrs | `19.3.0` |  | 9.4μs
+valideer | `0.4.2` | 1.4x slower | 12.9μs
+trafaret | `2.0.2` | 7.1x slower | 66.5μs
+pydantic | `1.5.1` | 8.7x slower | 81.3μs
```

Comment:
I'm assuming the benchmarks are meant to be relative to `pydantic` no? Since it's about comparing `pydantic` to other things?

Reply:
They're meant to be relative to the fastest, which up until now has been `pydantic`.